### ISSUE TITLE
[fr] verbes + préposition

### ISSUE THREAD

**Issue by Rebecca-auque** [languagetool-org/languagetool#5469]
This is no rule in order to suggest the right preposition after a verb. 
Je l'encourage de venir rapidement. -> à venir
Elle est intéressée à visiter la ville-> par une visite
Il a accepté à venir demain.->de venir

**Comment by udomai** [comment]
Je l'**encourage de** surcroît à faire des sports extrêmes.
Il n'est **intéressé à** vrai dire que par les voitures de sport.
Elle a **accepté à** ma plus grande surprise de venir demain.

**Comment by matthiastummers** [comment]
@udomai, I think you are missing punctuation.
What comes after what you added is identical to what @Rebecca-auque proposes.

Je l'**encourage**, de surcroît, **à** faire des sports extrêmes.
Il n'est **intéressé**, à vrai dire, uniquement **par** les voitures de sport.
Elle a **accepté**, à ma plus grande surprise, **de** venir demain.

**Comment by udomai** [comment]
@matthiastummers, that is true in theory. But what if the user makes that comma error? It would still be a false positive by Rebecca's preposition rule. That is why we need antipatterns for sentences like those I mentioned in my previous comment.
